doc/changes/latest.rst

```diff
@@ -37,6 +37,8 @@ Enhancements
 
 - :func:`~glm.first_level.first_level_from_bids` now takes an optional ``sub_labels`` argument and warns users of given subject labels that are not present in the dataset (:gh:`3351` by `Kevin Sitek`_).
 
+- :class:`NiftiMasker` now warns users if the data is being resampled to mask resolution (:gh:`3631` by `Mohammad Torabi`_).
```

Comment:
Thanks @ymzayek ! I wasn't sure what's the correct format of referencing things. Is there any instructions about it?

Reply:
With sphinx it's not always so clear so I wouldn't worry too much about it. I just know for linking within nilearn you need to first use the module name. The tilda is so that when it's rendered only the class name is printed. There are some guidelines also here: https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#